test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo-addon-analytic_operating_unit @ git+https://github.com/OCA/operating-unit.git@refs/pull/669/head#subdirectory=analytic_operating_unit
```

Comment:
The module `analytic_operating_unit`has been merged. Please update the test-requirements.txt file.